cypress.json

```diff
@@ -3,10 +3,15 @@
   "requestTimeout": 60000,
   "responseTimeout": 60000,
   "baseUrl": "http://localhost:5601",
-  "reporter": "cypress-multi-reporters",
+  "reporter": "cypress-mochawesome-reporter",
   "reporterOptions": {
-    "configFile": "reporter-config.json"
+    "reportDir": "cypress/reports",
```

Comment:
I would modify the config file: 
https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/main/reporter-config.json